src/components/Form.js

```diff
@@ -20,21 +21,26 @@ const Form = () => {
     document.getElementById('form').reset();
   };
   return (
-    <form onSubmit={handleSubmit} id="form">
-      <input type="text" onChange={(e) => setTitle(e.target.value)} name="title" placeholder="Book Title" required />
-      <input type="text" onChange={(e) => setAuthor(e.target.value)} placeholder="Author" required />
-      <select
-        type="text"
-        value={category}
-        onChange={(e) => setCategory(e.target.value)}
-        placeholder="category"
-      >
-        <option value="Leadership">Leadership</option>
-        <option value="Science">Science</option>
-        <option value="Fiction">Fiction</option>
-      </select>
-      <button type="submit">Add Book</button>
-    </form>
+    <div className={styles.container_form}>
+      <div className={styles.line} />
+      <h2 className={styles.container_form_h2}>Add New Book</h2>
+      <form onSubmit={handleSubmit} className={styles.container_form_form} id="form">
+        <input type="text" className={styles.container_form_form_input1} onChange={(e) => setTitle(e.target.value)} name="title" placeholder="Book Title" required />
+        <input type="text" className={styles.container_form_form_input1} onChange={(e) => setAuthor(e.target.value)} placeholder="Author" required />
+        <select
+          type="text"
+          className={styles.container_form_form_input2}
+          value={category}
+          onChange={(e) => setCategory(e.target.value)}
+          placeholder="category"
+        >
+          <option value="Leadership">Leadership</option>
+          <option value="Science">Science</option>
+          <option value="Fiction">Fiction</option>
+        </select>
+        <button type="submit" className={styles.container_form_form_button}>Add Book</button>
+      </form>
+    </div>
```

Comment:
- [ ] Please make sure that all form elements including the add button are on a single row as on the Zeplin design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/201099246-ff970628-4ae0-4e04-9d6c-8e28074129a0.png)

### Zeplin design
![image](https://user-images.githubusercontent.com/83803414/201099157-4fa6aa7f-9ed4-4475-a451-50aad1fbfde1.png)

Reply:
thank you